### ISSUE TITLE
Android App crashes on sign in

### ISSUE THREAD

**Issue by jdeniau** [lesspass/lesspass#416]
Hi,

When login in on the latest Android app, the app crashes and closes every time.

It worked fine with older version of the app.

App version : 3.1.5

**Comment by jdeniau** [comment]
In order to debug, here is the return of `adb logcat`:

```
04-15 16:45:12.546  1410  1540 D ActivityTrigger: activityResumeTrigger: The activity in ApplicationInfo{ba45af8 com.lesspass.android} is now in focus and seems to be in full-screen mode
04-15 16:45:12.546  1410  1540 E ActivityTrigger: activityResumeTrigger: not whiteListedcom.lesspass.android/com.lesspass.MainActivity/30105
04-15 16:45:32.618 13429 13457 E AndroidRuntime: Process: com.lesspass.android, PID: 13429
04-15 16:45:32.621  1410  4154 W ActivityManager:   Force finishing activity com.lesspass.android/com.lesspass.MainActivity
04-15 16:45:32.650  1410  2164 W InputDispatcher: channel 'd66f8e6 com.lesspass.android/com.lesspass.MainActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
04-15 16:45:32.651  1410  2164 E InputDispatcher: channel 'd66f8e6 com.lesspass.android/com.lesspass.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
04-15 16:45:32.658  1410  4154 I WindowManager: WIN DEATH: Window{d66f8e6 u0 com.lesspass.android/com.lesspass.MainActivity}
04-15 16:45:32.658  1410  3423 I ActivityManager: Process com.lesspass.android (pid 13429) has died: vis  +99TOP 
04-15 16:45:32.658  1410  4154 W InputDispatcher: Attempted to unregister already unregistered input channel 'd66f8e6 com.lesspass.android/com.lesspass.MainActivity (server)'
04-15 16:45:32.667   789   888 W SurfaceFlinger: Attempting to set client state on removed layer: com.lesspass.android/com.lesspass.MainActivity#0
04-15 16:45:32.668   789   888 W SurfaceFlinger: Attempting to destroy on removed layer: com.lesspass.android/com.lesspass.MainActivity#0
04-15 16:45:32.774  3582  3582 W View    : requestLayout() improperly called by com.google.android.apps.gsa.staticplugins.nowstream.shared.renderers.fe{bfcf603 V.E...... ......ID 0,0-1440,2560} during layout: running second layout pass
04-15 16:45:32.809  3582  3582 W View    : requestLayout() improperly called by com.google.android.apps.gsa.staticplugins.nowstream.shared.renderers.fe{bfcf603 V.E...... ......ID 0,0-1440,2560} during layout: running second layout pass
04-15 16:45:33.069 14131 14154 I StabilityMonitor: [ParserUtility] key word : com.lesspass.android
04-15 16:45:33.071 14131 14154 I StabilityMonitor: [APRReport] exceptionItem  tag:data_app_crash packageName:com.lesspass.android versionCode:30105 versionName:3.1.5
04-15 16:45:33.096 14131 14154 E STBIntentService: send error message:data_app_crash;1555339532630(pkgName:com.lesspass.android;version:3.1.5;versionCode:30105) to BBS for ANR/FC/TOMB_STONE...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1554915994262.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1554916006485.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1554916026887.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_native_crash@1554923215384.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1554923248788.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555267051115.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_native_crash@1555335864989.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.101 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555335879194.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555335915882.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555336009614.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555336032352.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/SYSTEM_TOMBSTONE@1555339005505.txt.gz;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/SYSTEM_TOMBSTONE@1555339005531.txt.gz;30105;3.1.5;com.lesspass.android into regularQue queue...
04-15 16:45:33.102 14131 14154 E STBMonitorQueue: restore data:/data/system/dropbox/data_app_crash@1555339282709.txt;30105;3.1.5;com.lesspass.android into regularQue queue...
```

I do not think it is really usefull then 🤔

**Comment by guillaumevincent** [comment]
Fix is coming :)

**Comment by guillaumevincent** [comment]
Fixed on version 3.1.6 on beta

**Comment by jdeniau** [comment]
Fix is working 😉

Thank you
